cg-spec/editors_draft.html

```diff
@@ -2724,18 +2753,50 @@ <h2>Referential opacity</h2>
+
+      <p>Referentially opaque semantics of embedded triples has unfortunate effects on common data integration procedures. As the following example illustrates it puts annotated statements at risk of fragmentation.</p>
+
+      <pre id="ref-opacity-owl-entailment-2"
+           data-transform="updateExample"
+           data-content-type="text/turtle"
+           class="nohighlight example"
+      >
+        <!--
+        #### under OWL-entailment
+
+        :car owl:sameAs https://www.wikidata.org/wiki/Q1420.
+        << :alice :buys :car >> :documentedBy :ellen.
+
+        #### does NOT entail
+
+        << :alice :buys https://www.wikidata.org/wiki/Q1420 >> :documentedBy :ellen.
+
+        #### Therefore an annotated statement construct
+
+        :alice :buys :car.
+        << :alice :buys :car >> :documentedBy :ellen.
+
+        #### after a standard data integration procedure may fall apart
+
+        :alice :buys https://www.wikidata.org/wiki/Q1420.
+        << :alice :buys :car >> :documentedBy :ellen.
+        -->
+        </pre>
+
+
+      <p>Note that the current semantics does not forbid the additional inferences to be drawn. However, they would require additional knowledge and an appropriate <a data-cite="RDF11-MT#dfn-semantic-extension">semantic extension</a>, not specified in this document. How such a semantic extension might be defined is a topic of ongoing <a href="https://github.com/w3c/rdf-star/issues/170">discussion</a>. So far no convincing solution has been presented. This makes RDF-star not readily usable for (but not incompatible with) the majority of use-cases.</p>
+        <p>These side effects of referentially opaque semantics will have to be clearly and pro-actively communicated to prevent unpleasant surprises from "missing" entailments and broken annotations.</p>
 
       </section>
 
       <section>
         <h2>Alternatives to referential opacity</h2>
 
-        <p>Given the potential drawbacks of the referential opacity provided by the current semantics, as described in the previous section, the group has considered alternative semantics. No clear consensus was reached, however, to decide whether these alternatives should be used as a replacement, or as a <a data-cite="RDF11-MT#dfn-semantic-extension">semantic extension</a>, of the current semantics.</p>
+      <p>Given the potential drawbacks of the referential opacity provided by the current semantics, as described in the previous section, the group has considered to provide embedded triples with RDF's standard referentially transparent semantics. No clear consensus was reached, however, to decide whether this alternative should be used as a replacement, or as a yet to be defined <a data-cite="RDF11-MT#dfn-semantic-extension">semantic extension</a>, of the current semantics.</p>
```

Comment:
```suggestion
      <p>Given the potential drawbacks of the referential opacity of quoted triples provided by the current semantics, as described in the previous section, the group has considered alternative semantics, including possibly treating all embedded triples as referentially transparent (i.e., as asserted) wherever they occur. No clear consensus was reached on whether this or any other alternative should be used as a replacement, or as a yet-to-be-defined <a data-cite="RDF11-MT#dfn-semantic-extension">semantic extension</a>, of the current semantics.</p>
```

Reply:
Referential opacity or transparency of embedded triples has nothing to do with the question if a triple (that the embedded triple represents) is asserted or not. These are orthogonal issues. Maybe check [example 35](https://pr-preview.s3.amazonaws.com/w3c/rdf-star/185/524a524...rat10:5f77b06.html#ref-opacity-owl-entailment-2), or our recent discussion in related issues.